libr/core/cmd.c

```diff
@@ -32,9 +32,9 @@
 //       whitespaces, because we let cmd_substitution_arg create
 //       new arguments
 static const char *SPECIAL_CHARS_REGULAR = "@;~$#|`\"'()<>";
-static const char *SPECIAL_CHARS_PF = "@;~$#|`\"'<>";
 
 #if USE_TREESITTER
+static const char *SPECIAL_CHARS_PF = "@;~$#|`\"'<>";
```

Comment:
There is no need to move it here, or move `SPECIAL_CHARS_REGULAR` too.

Reply:
this was because when you compile without tree-sitter, compiler complains about SPECIAL_CHARS_PF not being used. SPECIAL_CHARS_REGULAR, instead, is used even without tree-sitter AFAIK. But please @radare , don't do this change here. I will commit a separate PR for this.

Reply:
yes, thats because its unused witohut treesitter. pls commit this as a separate PR, otherwise ill do it today , as well as ill do with the rest of unrelated/random changes in this pr